assets/locales/en.json

```diff
@@ -1,9 +1,10 @@
 {
-  "ipfsIsRunning": "IPFS is running",
-  "ipfsIsStopping": "IPFS is stopping",
-  "ipfsIsStarting": "IPFS is starting",
-  "ipfsIsNotRunning": "IPFS is not running",
-  "ipfsHasErrored": "IPFS has errored",
+  "ipfsIsRunning": "IPFS is Running",
+  "ipfsIsStopping": "IPFS is Stopping",
+  "ipfsIsStarting": "IPFS is Starting",
+  "ipfsIsNotRunning": "IPFS is Not Running",
+  "ipfsHasErrored": "IPFS has Errored",
+  "runningWithGC": "Running (Garbage Collecting)",
```

Comment:
> (drive by comment, will do proper review on Monday)

Not sure why, but below reads more "natural" to me:

```suggestion
  "runningWithGC": "Running (GC in progress…)",
```

Thoughts @jessicaschilling?

Reply:
The biggest concern might be if the user knows or not what 'GC' means. However, if they're running the GC, it's highly likely that they already knew what it was and does.

Reply:
Agreed that it seems more natural. And I think what @hacdias brings up about putting "(GC)" in the preceding dialog will help considerably.

The only thing about this plan I would change is remove the "..." in the "Running" item, because in the context of menu items that implies "choose this menu item and then we give you a dialog with more info".

Reply:
Just updated it! 

![image](https://user-images.githubusercontent.com/5447088/79792247-79a49500-8346-11ea-8c13-ca9283c1f0f0.png)

![image](https://user-images.githubusercontent.com/5447088/79792263-81643980-8346-11ea-8f35-b11e27260966.png)

Reply:
I'm down if @lidel is. 😊